tests/analysis_tests/tests/src/expected/CompletionTypeT.res.txt

```diff
@@ -9,22 +9,22 @@ Path date
     "kind": 12,
     "tags": [],
     "detail": "Js.Date.t",
-    "documentation": null
+    "documentation": {"kind": "markdown", "value": "\nA type representing a JavaScript date.\n"}
   }, {
     "label": "Some(_)",
     "kind": 12,
     "tags": [],
     "detail": "Js.Date.t",
-    "documentation": null,
+    "documentation": {"kind": "markdown", "value": "\nA type representing a JavaScript date.\n"},
     "insertText": "Some(${1:_})",
     "insertTextFormat": 2
   }, {
-    "label": "Some(date)",
+    "label": "Some(stdlib_Date)",
```

Comment:
@zth Weird that this starts with a lower case letter here.

Reply:
Well, it's an identifier so it's not weird per se. But, we should probably filter out the `stdlib` part.